packages/neutron-sdk/src/interchain_queries/v047/queries.rs

```diff
@@ -0,0 +1,77 @@
+// import all queries from v045 package
+// to make it available from v047 package (kinda proxy) since they work with Cosmos SDK 0.47 as usual
+pub use crate::interchain_queries::v045::queries::*;
+
+// But at the same time we replace v045::BalancesResponse and v045::query_balance() with
+// corresponding structure and method below to create a structure and a method
+// compatible with Cosmos SDK 0.47
```

Comment:
> replace v045::BalancesResponse and v045::query_balance()

there are also DelegatorDelegationsResponse and query_delegations() replaced below. I think being detailed here is redundant. let's just say here that some stuff is redefined below without any specific methods and structs listed? otherwise comment can become outdated in the future

Reply:
Fixed